db/seeds/index.js

```diff
@@ -1,10 +1,18 @@
 'use strict';
 
-const Movies = require('./data/movies');
+const Movies    = require('./data/movies');
+const Locations = require('./data/locations');
 
 exports.seed = function (Knex) {
-  return Knex('movies').truncate()
+  return Promise.all([
+    Knex('movies').del(),
+    Knex('locations').del()
```

Comment:
Nit: usually when we want to delete a whole table, we use `.truncate` vs `.del`. [There aren't too many differences](https://stackoverflow.com/questions/139630/whats-the-difference-between-truncate-and-delete-in-sql), but it seems to be a bit faster and it follows the convention we have elsewhere.